Update kbn.js - correct typo for megabits/sec

marknmel opened the pull request:
data rate for Mbits has typo - should be megabits/sec - not megabites/sec

marknmel committed:
Update kbn.js

data rate for Mbits has typo - should be megabits/sec - not megabites/sec

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=4083) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

utkarshcmu commented on public/app/core/utils/kbn.js:
```diff
@@ -595,7 +595,7 @@ function($, _) {
           {text: 'bytes/sec',   value: 'Bps'},
           {text: 'kilobites/sec', value: 'Kbits'},
```
Kilobits :)

utkarshcmu commented on public/app/core/utils/kbn.js:
```diff
@@ -595,7 +595,7 @@ function($, _) {
+          {text: 'megabits/sec', value: 'Mbits'},
           {text: 'megabytes/sec',    value: 'MBs'},
           {text: 'gigabytes/sec',   value: 'GBs'},
           {text: 'gigabites/sec',   value: 'Gbits'},
```
Gigabits

marknmel commented:
I should have had another coffee :-) - thanks!

marknmel committed:
Updated typos - Kilobits, Megabits and Gigabits

Update typos - Kilobits, Megabits and Gigabits - not Kilobites, Megabites, or Gigabites

utkarshcmu commented:
:+1:

bergquist commented:
@marknmel thanks for spotting this and helping out :)
